src/preferences/dialog/dlgprefsounddlg.ui

```diff
@@ -66,6 +66,20 @@
       <widget class="QComboBox" name="deviceSyncComboBox"/>
      </item>
      <item row="4" column="0">
+      <widget class="QLabel" name="engineClockLabel">
+       <property name="text">
+        <string>Engine Clock</string>
+       </property>
+      </widget>
+     </item>
+     <item row="4" column="1">
+      <widget class="QComboBox" name="engineClockComboBox">
+       <property name="toolTip">
+        <string>Use soundcard clock for low latency setups. Use network clock for broadcasting only setups.</string>
```

Comment:
This should explain better in what situations this option might be useful. What is a "broadcasting only setup"? When there is no soundcard in use at all? If a user selects this option while broadcasting and monitoring the Master output on a soundcard, is there a chance of audible artifacts on the Master output?

Reply:
Broadcasting is just the same as a second soundcard with an individual clock. 
Normally, the master output is taken to drive the mixxx engine,which produces a bit perfect output on master, all other soundcards (hardware and broadcasting) suffer from stuffing or lost samples. 
This is unbearable in case of a hardware soundcard, but can be heard in broadcasting, because the big buffer sizes. 
If you use the Network clock from this PR master will suffer also from unhearable stuffing or lost samples, but Broadcasting is bit perfect then assuming that the receiver runs also on an NTP synced cock.
With broadcast only I mean, that the audience is listening to broadcast only. No one cares about the quality of the master output in this case. 

Do you have a better tooltip in mind? 
"Use the network clock if the primary output is broadcasting"

Reply:
How about: "Use soundcard clock for live audience setups. Use network clock for broadcasting without a live audience." In the manual we can explain the reasoning.

Reply:
An additional critical point is, that we need some extra latency for the syncing code.
On my system I suffer master crackling at <= 10 ms buffer size.  

How about this: 
"Use soundcard clock for live audience setups and lowest latency. Use network clock for broadcasting without a live audience."